ase/io/espresso_cp.py

```diff
@@ -73,6 +73,7 @@
 def write_espresso_cp_in(fd, atoms, input_data=None, pseudopotentials=None,
```

Comment:
and then I would change this to not take the arguments `kspacing` and `kpts` and `koffset`

Reply:
Sounds good!